api/internal/controllers/analysis/analysis.go

```diff
@@ -147,11 +158,90 @@ func (c *Controller) hasDuplicatedHash(
 }
 
 func (c *Controller) publishInBroker(analysisID uuid.UUID) error {
-	response, err := c.GetAnalysis(analysisID)
+	res, err := c.GetAnalysis(analysisID)
 	if err != nil {
 		return err
 	}
 
 	return c.broker.Publish("", exchange.NewAnalysis,
-		exchange.Fanout, response.ToBytes())
+		exchange.Fanout, res.ToBytes())
+}
+
+//TODO:REMOVE ALL BELOW AFTER v2.10.0
```

Comment:
I think that would be good add a documentation here, describing the method and why we will remove soon.